src/collision/collision_object.cpp

```diff
@@ -27,6 +27,7 @@ CollisionObject::CollisionObject(CollisionGroup group, CollisionListener& listen
   m_group(group),
   m_movement(0.0f, 0.0f),
   m_dest(),
+  m_unisolid(false),
```

Comment:
No need for 'false' here, it's the default anyway.

Reply:
Usually I prefer to specify, so you can see it's the default. But I can remove it too.

Reply:
fine.